packages/config-express-middleware/src/index.ts

```diff
@@ -1,4 +1,4 @@
-import express from 'express'
+import express from 'express-serve-static-core'
```

Comment:
Do we have to change any dependencies in package.json

Reply:
It's being pulled in already